src/redux/actions/constants.js

```diff
@@ -0,0 +1,4 @@
+export const details={
+    user:null,
+    isVerified:false,
+}
```

Comment:
shift this to inside slice and declare like const initialState=... and use this in slice.
we don't need actions folder in toolkit for now